.gitignore

```diff
@@ -0,0 +1,6 @@
+data/panel/*
```

Comment:
i generally wouldn't recommend modifying the `.gitignore` in such contexts but as thi sis specific to your branch I'll ignore this ;)